### PR TITLE
Circleci pack

### PR DESCRIPTION
**Pull request by manasdk**
### Status 
- pack extension, add new commands

### Description

Add 2 new actions. 
* `get_build_info`
* `run_build`

### Checklist (tick everything that applies)

- [x] Metadata: pack.yaml, icon, structure (required for new packs)
- [x] Version bump (required for changed packs)
- [x] Code linting (required, can be done after the PR checks)
- [ ] Tests (not required but really recommended)

### CODE REVIEW

File: packs/circle_ci/actions/run_build.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+runner_type: run-python
+description: Run build for project and branch.
+enabled: true
+entry_point: run_build.py
+parameters:
+  project:
+    type: string
+    description: Name of project in circle ci.
+    required: true
+  branch:
```
Comment (Kami) [inline packs/circle_ci/actions/run_build.yaml]: Can it just be a branch or any valid SHA / symbolic ref?

Comment (manasdk) [inline packs/circle_ci/actions/run_build.yaml]: Their API is really awkward but atleast there is an API doc. I am still looking at the details to see what is possible.

There is support for SHA and tag but the URL is different.

Comment (Kami) [inline packs/circle_ci/actions/run_build.yaml]: I see, I'm fine with this approach for now then.

Comment (manasdk) [inline packs/circle_ci/actions/run_build.yaml]: Added ability to use `tag` and `vcs_revision`

**Comment by manasdk** [comment]
No aliases yet likely those will be added when the CI pipeline is made chatops friendly.

**Pull request merged**